src/devices/Mfrc522/Mfrc522.cs

```diff
@@ -296,8 +296,9 @@ public bool ListenToCardIso14443TypeA(out Data106kbpsTypeA card, TimeSpan timeou
         /// Check if a new card is present.
         /// </summary>
         /// <param name="atqa">ATQA buffer must be 2 bytes length and will contain the ATQA answer if there is a card.</param>
+        /// <param name="reselect">true if this is reselecting an existing card (e.g., after a halt)</param>
         /// <returns>true if there is a card, else false.</returns>
-        public bool IsCardPresent(byte[] atqa)
+        public bool IsCardPresent(byte[] atqa, bool reselect = false)
```

Comment:
NIT: In order to not be binary breaking, any reason why we shouldn't just add new overloads as opposed to using default parameters?